x/emissions/keeper/keeper.go

```diff
@@ -438,25 +429,6 @@ func (k *Keeper) GetParamsRewardCadence(ctx context.Context) (int64, error) {
 	return params.RewardCadence, nil
 }
 
-// Gets the total sum of all stake in the network across all topics
-func (k *Keeper) GetTotalStake(ctx context.Context) (Uint, error) {
```

Comment:
I question why we're deleting this, I would think this is something we would want to expose via query even if we don't use it ourselves